jbpm-bpmn2/src/main/java/org/jbpm/bpmn2/xml/IntermediateThrowEventHandler.java

```diff
@@ -325,6 +331,7 @@ protected void readDataInputAssociation(org.w3c.dom.Node xmlNode,
             subNode = subNode.getNextSibling();
             if (subNode != null && "transformation".equals(subNode.getNodeName())) {
                 String lang = subNode.getAttributes().getNamedItem("language").getNodeValue();
+                if (text.startsWith("\"") && text.endsWith("\"")) {
```

Comment:
Compilation error, missing closing bracket:
[ERROR] /home/jenkins/workspace/KIE/main/pullrequest/jbpm-main.pr/bc/kiegroup_jbpm/jbpm-bpmn2/src/main/java/org/jbpm/bpmn2/xml/IntermediateThrowEventHandler.java:[382,9] illegal start of expression